client/app/components/Header/__tests__/HeaderProfile.spec.jsx

```diff
@@ -0,0 +1,96 @@
+// @flow
+import { queryByText, render } from '@testing-library/react';
+
+import { HeaderProfile } from '../HeaderProfile';
+import React from 'react';
+
+const profileWithAvatar = { 
+  avatar: '//congue.volutpat/sollicitudin.jpg',
+  name: 'Sollicitudin',
+  profile: {
+    name: 'Name One',
+    url: '/some-path',
+    active: true,
+    dataMethod: 'delete',
+    hideInMobile: false,
+  },
+  account: {
+    name: 'Name Two',
+    url: '/some-other-path',
+    active: true,
+    dataMethod: 'delete',
+    hideInMobile: false,
+  },
+  notifications: {
+    plural: 'plural',
+    none: 'none',
+    clear: 'clear',
+  },
+};
+
+const profileWithoutAvatar = { 
+  name: 'Volutpat',
+  profile: {
+    name: 'Name One',
+    url: '/some-path',
+    active: true,
+    dataMethod: 'delete',
+    hideInMobile: false,
+  },
+  account: {
+    name: 'Name Two',
+    url: '/some-other-path',
+    active: true,
+    dataMethod: 'delete',
+    hideInMobile: false,
+  },
+  notifications: {
+    plural: 'plural',
+    none: 'none',
+    clear: 'clear',
+  },
+};
+
+describe('HeaderProfile', () => {
+  describe('has no avatar', () => {
+    it('renders correctly', () => {
+      const { container } = render(<HeaderProfile profile={profileWithoutAvatar} />);
+      const headerProfileSection = container.querySelector('.headerProfile');
+      const headerProfileInfoSection = container.querySelector('.headerProfile');
+      const avatarSection = container.querySelector('.avatar');
+      const buttonGhostXSSection = container.querySelector('.buttonGhostXS');
+      expect(headerProfileSection).toBeInTheDocument();
+      expect(headerProfileInfoSection).toBeInTheDocument();
+      expect(avatarSection).toBeInTheDocument();
+      expect(buttonGhostXSSection).toBeInTheDocument();
+      expect(container).not.toBeNull();
```

Comment:
You can get rid of this expect and the one after. Your other expects passing implies that these ones are passing.